bin/test_integration_rhel8

```diff
@@ -35,7 +35,6 @@ function _random_password() {
 }
 
 echo "Configuring Conjur..."
-export ADMIN_PASSWORD=$(_random_password)
```

Comment:
you can also get rid of `_random_password` function since this is the only place we use it